ccdiManifestURL.json

```diff
@@ -1,29 +1,30 @@
 {
   "phs000466": {
-    "TARGET_CCSK_ClinicalData_Discovery_20170525.xlsx": "https://d2l5jy2ao2mx5b.cloudfront.net/TARGET-CCSK_clinicalData_PUBLIC_20220207/TARGET_CCSK_ClinicalData_Discovery_20170525.xlsx",
+    "TARGET_CCSK_ClinicalData_Discovery_20170525.xlsx": "https://d2l5jy2ao2mx5b.cloudfront.net/target/phs000466/TARGET_CCSK_ClinicalData_Discovery_20170525.xlsx",
     "isTarget": true
   },
   "phs000467": {
-    "TARGET_NBL_ClinicalData_Discovery_20220125.xlsx": "https://d2l5jy2ao2mx5b.cloudfront.net/target-nbl/TARGET_NBL_ClinicalData_Discovery_20220125.xlsx",
-    "TARGET_NBL_ClinicalData_Validation_20220125.xlsx": "https://d2l5jy2ao2mx5b.cloudfront.net/target-nbl/TARGET_NBL_ClinicalData_Validation_20220125.xlsx",
+    "TARGET_NBL_ClinicalData_Discovery_20220125.xlsx": "https://d2l5jy2ao2mx5b.cloudfront.net/targe/phs000467/TARGET_NBL_ClinicalData_Discovery_20220125.xlsx",
+    "TARGET_NBL_ClinicalData_Validation_20220125.xlsx": "https://d2l5jy2ao2mx5b.cloudfront.net/targe/phs000467/TARGET_NBL_ClinicalData_Validation_20220125.xlsx",
```

Comment:
target misspelled

Reply:
[qiongL910](https://github.com/qiongL910) Can you resolve this typo?